getting-started/enable-debug-logging.md

```diff
@@ -31,7 +31,7 @@ MobileCore.setLogLevel(LoggingMode.DEBUG);
 ```
 {% endtab %}
 
-{% tab title="iOS - Swift" %}
+{% tab title="iOS (AEP 3.x)" %}
 ## Swift
 
 ```swift
```

Comment:
The 3.x examples seem to be incorrect, should use MobileCore (Swift) and AEPMobileCore (Objc). We also use .trace instead of verbose in this version of the SDK.